packet/BGPConfig.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: LGPL-3.0-only
+
+
+class BGPConfig:
```

Comment:
Does packet-python must be 2.7-compatible? If not, it would be good to use dataclasses for the API resource types. It would make the code much cleaner, and it would be more type-safe.
https://www.reddit.com/r/learnpython/comments/9h74no/convert_dict_to_dataclass/

Reply:
Most Python SDKs support version 2.7, and I don't see any extra effort needed to support it. So I would leave it.

Reply:
OK